docker/src/main/java/brooklyn/entity/container/docker/DockerInfrastructure.java

```diff
@@ -133,6 +133,11 @@
             new TypeToken<Map<String, Object>>() { },
             "docker.dockerfile.substitutions", "Dockerfile template substitutions", MutableMap.<String, Object>of());
 
+    @SetFromFlag("shouldStartRegistry")
```

Comment:
Make this a catalog entry property, so we can see it when we start things from the catalog.